docs/UserGuide.md

```diff
@@ -1,44 +1,85 @@
 ---
 layout: page
 title: AcademyAssist User Guide
+
 ---
+# AcademyAssist User Guide
+
+## Table of Contents
+
+1. [Introduction](#introduction)
+
+2. [Quick Start](#quick-start)
+
+3. [Features](#features)
+   * [Adding a student](#adding-a-student--add)
+   * [Deleting a student](#deleting-a-student--del)
+   * [Editing a student](#editing-a-student--edit)
+   * [Viewing all student](#viewing-all-students--view)
+   * [Finding a student](#finding-a-student--find)
+   * [Sorting students](#sorting-students--sort)
+   * [Clearing all entries](#clearing-all-entries--clear)
+   * [Viewing help](#viewing-help--help)
```

Comment:
Maybe can change this to Getting help? It might be a bit confusing with viewing all student above since its similar format but different functionality